libminifi/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ endif()
 file(GLOB PROCESSOR_SOURCES  "src/processors/*.cpp" )
 
 
-add_library(core-minifi SHARED ${SOURCES})
+add_library(core-minifi SHARED ${SOURCES} include/utils/net/AsioSocketUtils.h)
```

Comment:
Is include/utils/net not included in the include directories or why is it needed to be added to the sources?

Reply:
that was probably my IDE automatically adding it, thanks for catching. removed it in https://github.com/apache/nifi-minifi-cpp/pull/1598/commits/3d451e15619339b1f54d65ee31f3c000a4f4e25f#diff-384b8bf2d8b6601d04935ea6bc051dee880b128f5add4fc99b66ca0f6a7bccbaL81